.github/workflows/release.yaml

```diff
@@ -19,6 +22,22 @@ jobs:
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Login to GitHub container registry
+        run: |
+          REPO_OWNER=`echo "${{ github.repository_owner }}" | tr '[:upper:]' '[:lower:]'`
+          echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
```

Comment:
Use the `docker/login-action@v2` action.

See: https://github.com/docker/login-action

Reply:
@tuler we don't have `DOCKER_USERNAME` nor `DOCKER_PASSWORD` for this repository

Should I use my own or use the onre already used on other repositories like `cartesi/rollups`?

Would you mind defining it?

Reply:
Done

Reply:
> docker/login-action@v2

Fixed !